### PR TITLE
Remove delete league icon

### PR DESCRIPTION
**Pull request by paulywill**
This is to finish resolving this issue: #48

I have commented out the link for the Delete League icon in the setting tab.

There still should be some clarification has to how admins will rename and delete leagues as part of housekeeping within the application.

### CODE REVIEW

**Comment by paulywill** [comment]
@makkoli in response to your answer in: https://github.com/freeCodeCamp/league-for-good/issues/48

Shall I change the commented out to actually deleting the lines?

File: client/src/components/dashboard/leagueTabs/leagueTabData.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import React from 'react';
 import AddIcon from 'material-ui/svg-icons/content/add';
 import AddPersonIcon from 'material-ui/svg-icons/social/person-add';
-import DeleteForeverIcon from 'material-ui/svg-icons/action/delete-forever';
+// import DeleteForeverIcon from 'material-ui/svg-icons/action/delete-forever';
```
Comment (paulywill) [inline client/src/components/dashboard/leagueTabs/leagueTabData.js]: commented out definition to pass Travis CI.

File: client/src/components/dashboard/leagueTabs/leagueTabData.js [hunk 2]
```diff
@@ -80,13 +80,13 @@ const settingsLinks = [
...
 		icon: <AddPersonIcon />,
 		url: LINK.SETTINGS_ADD_STAFF_FORM
-	},
-	{
-		description: 'Delete your league.',
-		label: 'DeleteLeague',
-		icon: <DeleteForeverIcon />,
-		url: LINK.SETTINGS_DELETE_LEAGUE
 	}
+//	{
```
Comment (paulywill) [inline client/src/components/dashboard/leagueTabs/leagueTabData.js]: Just commented out until delete was explained.

These lines can be completely removed.

**Comment by makkoli** [comment]
@paulywill The only way to delete leagues will be for whoever is maintaining the database to go in and delete it. I do like the rename idea, and if you want to work on it that'd be great.

**Pull request merged**